java/src/Ice/src/main/java/com/zeroc/Ice/PluginManagerI.java

```diff
@@ -6,7 +6,7 @@
 
 import java.net.URLEncoder;
 
-public final class PluginManagerI implements PluginManager {
```

Comment:
This was one of the `...I` files that was already in `Ice`.
I made it package private, since all the `...I` files should be purely internal.
That's fine right?

Reply:
Yes, all ...I classes should be package-private.